tests/scripts/check-python-files.sh

```diff
@@ -74,7 +74,7 @@ $PYTHON -m pylint -j 2 scripts/mbedtls_dev/*.py scripts/*.py tests/scripts/*.py
 }
 
 # Check types if mypy is available
-if type mypy >/dev/null 2>/dev/null; then
+if can_mypy; then
```

Comment:
I am wondering why we don't have the same kind of test for pylint.

Reply:
Pylint was mandatory before, and I wanted to keep it mandatory. I'm not making mypy mandatory in this PR because it isn't available on Jenkins yet. I'm open to suggestions on how to do this differently.

Reply:
That's fine by me to keep Pylint checks mandatory.